tasklist/client/src/modules/hooks/usePermissions.tsx

```diff
@@ -13,10 +13,12 @@ const usePermissions = (scopes: Permissions) => {
   const {data: currentUser} = useCurrentUser();
 
   const permissions = currentUser?.permissions;
+  if (currentUser && permissions === undefined) {
```

Comment:
```suggestion
  const permissions = currentUser?.permissions;
  if (permissions === undefined) {
```

Reply:
@megglos On the new Identity we won't have the permissions property on the UserDTO like we used to have?

Reply:
ah got it, missed to revert that part too, will do

Reply:
yes, those permissions will be gone, we will only have application ACCESS, no read/write differentiation - that will be resource based insteat, see [here](https://docs.google.com/document/d/1ikxw9ARCv-MBwASnj7WhW5uHCndCRk2B35xi-cB2N0M/edit?tab=t.0#heading=h.c0kx9z3moo6t)

Reply:
so effectively we could remove that property from all mocks and typedefs you mentioned here, right?
https://github.com/camunda/camunda/pull/28134#issuecomment-2668716755

Reply:
handled with https://github.com/camunda/camunda/pull/28134/commits/581246ad6afd49d50c3f132ea8d67a43f1725a7d by @vsgoulart , thank you!